File: app/src/main/java/in/woloo/www/giftSubscription/model/SendGiftCardResponse.java
package in.woloo.www.giftSubscription.model;

import com.google.gson.annotations.SerializedName;

public class SendGiftCardResponse{

	@SerializedName("code")
	private int code;

	@SerializedName("message")
	private String message;

	@SerializedName("order_id")
	private String orderId;

	@SerializedName("status")
	private String status;

	public int getCode(){
		return code;
	}

	public String getMessage(){
		return message;
	}

	public String getOrderId(){
		return orderId;
	}

	public String getStatus(){
		return status;
	}
}

File: app/src/main/java/in/woloo/www/more/giftSubscription/model/SendGiftCardResponse.kt
package `in`.woloo.www.more.giftSubscription.model

import com.google.gson.annotations.SerializedName

class SendGiftCardResponse {
    @SerializedName("code")
    val code: Int = 0

    @JvmField
	@SerializedName("message")
    val message: String? = null

    @JvmField
	@SerializedName("order_id")
    val orderId: String? = null

    @JvmField
	@SerializedName("status")
    val status: String? = null
}